.github/workflows/govulncheck.yaml

```diff
@@ -0,0 +1,32 @@
+# Copyright 2025 Defense Unicorns
+# SPDX-License-Identifier: AGPL-3.0-or-later OR LicenseRef-Defense-Unicorns-Commercial
+
+name: golangci-lint
+on:
+  push:
+    branches: ["main"]
+  pull_request:
+    paths-ignore:
+      - "**.md"
+      - "**.jpg"
+      - "**.png"
+      - "**.gif"
+      - "**.svg"
+      - "adr/**"
+      - "docs/**"
+      - "CODEOWNERS"
+      - "goreleaser.yml"
+
+permissions:
+  contents: read
+
+jobs:
+  govulncheck_job:
+    runs-on: ubuntu-latest
+    name: Run govulncheck
+    steps:
+      - uses: actions/checkout@v4
+      - name: Setup golang
+        uses: ./.github/actions/golang
+      - id: govulncheck
+        uses: golang/govulncheck-action@v1
```

Comment:
it would be totally reasonable to combine these two workflows into a single `go.yaml` for instance to reduce some of the duplication of the head matter (unless you see a universe where said head matter will later diverge meaningfully)

Reply:
I'm not sure what you mean here, and google isn't being helpful

Reply:
Combined workflows